index.html

```diff
@@ -0,0 +1,240 @@
+            a project you need coded, 
+            don’t hestiate to contact me.</p>
+        <h3>
+            <a href="">LET’S CONNECT</a> 
+        </h3>  
+        <ul class="social-media">
+            <li><a href="twiter"><img src="../Portfolio-finish-mobile-version/img/twiter.svg" alt=""></a></li>
+            <li><a href="linkedin"><img src="../Portfolio-finish-mobile-version/img/Linkedin.svg" alt=""></a></li>
+            <li><a href="medium"><img src="../Portfolio-finish-mobile-version/img/medium.svg" alt=""></a></li>
+            <li><a href="github"><img src="../Portfolio-finish-mobile-version/img/github.svg" alt=""></a></li>
+            <li><a href="chill"><img src="../Portfolio-finish-mobile-version/img/chill.svg" alt=""></a></li>
+        </ul> 
+      </div>
+    </section>
+    <main class="work">
+      <ul>
+      <li class="item tonic">
+        <img src="../Portfolio-finish-mobile-version/img/work1.svg" alt="">
+        <div>
+          <div>
+            <h1>Tonic</h1>
+            <ul class="canopy">
+              <li><h3>CANOPY</h3></li>
+              <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+              <li><p>Back End Dev</p></li>
+              <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+              <li><p>2015</p></li>
+            </ul>
+            </div>
+            <p>
+                A daily selection of privately
+                personalized reads; no accounts
+                or sign-ups required.
+            </p>
+            <ul class="languages">
+              <li><a href="">html</a></li>
+              <li><a href="">css</a></li>
+              <li><a href="">javascript</a></li>
+              <li><a href="">Ruby on rails</a></li>
+            </ul>
+            <div class="rev">
+              <button><a href="" class="btn">See Project</a></button>
+            </div>
+          </div>
+        </li>
+      <li class="item multi-post-stories">
+        <img src="../Portfolio-finish-mobile-version/img/work2.svg" alt="">
+        <div>
+          <div>
+          <h1>Multi-Post Stories</h1>
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
```

Comment:
- Let's also add the missing interaction between buttons by indicating when it's active, hovered, pressed, etc. kindly check the Figma design to see the correct color used to improve our design.